crypto/fipsmodule/rand/rand.c

```diff
@@ -351,14 +300,33 @@ void RAND_bytes_with_additional_data(uint8_t *out, size_t out_len,
       state = &stack_state;
     }
 
+#if defined(BORINGSSL_FIPS)
+    // Initialize the thread-local Jitter instance.
+    state->jitter_ec = NULL;
+    // The first parameter passed to |jent_entropy_collector_alloc| function is
+    // the desired oversampling rate. Passing a 0 tells Jitter module to use
+    // the default rate (which is 3 in Jitter v3.1.0).
```

Comment:
Can we verify the Jitter version in any way? At run-time, compile-time?

Reply:
we can, I'll add the check to the unit test